$NetBSD: patch-aa,v 1.2 2020/03/22 21:21:05 rillig Exp $

perl_install causes the Makefile in perl/ to be rebuilt with /usr/pkg/lib
as library path instead of the source tree. install then tries to relink
the target with the new path, which fails because the just-installed
libraries are not in the buildlink directory.
Just don't rebuild makefiles, the run-time search path has been fixed
by buildlink already.

--- glue/Makefile.am.orig	2011-03-29 21:44:52.000000000 +0200
+++ glue/Makefile.am	2011-03-29 21:45:29.000000000 +0200
@@ -15,7 +15,6 @@
 	cd perl; $(MAKE) test
 
 perl_install:
-	cd perl; INSTALL=1 @PERL@ @PERL_OPTS@ Makefile.PL -apxs @APACHE2_APXS@ @MM_OPTS@
 	cd perl; $(MAKE) install
 
 perl_clean:
